corehq/apps/userreports/expressions/getters.py

```diff
@@ -86,7 +86,10 @@ def transform_int(item):
     try:
         return int(item)
     except (ValueError, TypeError):
-        return None
+        try:
+            return int(float(item))
+        except (ValueError, TypeError):
+            return None
```

Comment:
We probably wouldn't run into this, but because `int()` falls back to using long ints if the number is too large for an integer, and `float()` doesn't, they can produce different results. e.g.
```

>>> v = "92233720368547758079" # sys.maxint with a 9 tacked on to the end
>>> int(v) == int(float(v))
False
>>> int(v)
92233720368547758079L
>>> int(float(v))
92233720368547758080L
>>> float(v)
9.223372036854776e+19
```
(I'm actually confused as to why `int(float(v))` doesn't return `92233720368547760000L` in this example)